pkg/controller/import-controller.go

```diff
@@ -656,32 +656,10 @@ func IsInsecureTLS(ep string, cdiConfig *cdiv1.CDIConfig, client client.Client,
 	}
 
 	for _, value := range cdiConfig.Spec.InsecureRegistries {
-		log.V(1).Info("Checking host against value", "host", url.Host, "value", value)
```

Comment:
Nothing will print if there are no insecure registries, but if there are some, and it is not working for some reason, having this will help debugging. We ran into that at some point, and we added the log because of it.

Reply:
Re-added the log